YasiTL/backpat
# Admin dash improvements

![PRGif](https://user-images.githubusercontent.com/14118065/123555495-a83cad80-d74b-11eb-93b6-67e5eb356a4c.gif)

# Test Plan
- Ensure visiting /admindashboard will bring up the appropriate page
- Check that tab switching works accurately
- Cross check data on firebase and on each tab and ensure the appropriate data appears
- Open and close the data rows
